pom.xml

```diff
@@ -1097,6 +1097,14 @@
                 <artifactId>alluxio-shaded-client</artifactId>
                 <version>${dep.alluxio.version}</version>
                 <exclusions>
+                    <exclusion>
```

Comment:
please ignore change in this file, just to make the build work with Alluxio client on snapshot versions.
once release version is cut, will revert the change

Reply:
These changes are not sufficient, it seems more work is needed to manage proper dependencies. The newer version is bringing incompatible version of many dependencies.

Reply:
The build in the PR as well is failing.